requirements.txt

```diff
@@ -1,3 +1,5 @@
 django>=1.8.7
-edx-django-oauth2-provider>=0.3.0,<1.0.0
+# edx-django-oauth2-provider>=0.3.0,<1.0.0
+# TODO update with new django-oauth2-provider release
+-e git+https://github.com/edx/django-oauth2-provider.git@d6bcdd9de8b3e0620b1bd7273ff5a3fe17fa80bd#egg=django-oauth2-provider
```

Comment:
I assume this editable install is only here until edx/django-oauth2-provider#22 lands?